src/views/QuestionBank/js/question-bank-detail.js

```diff
@@ -1,42 +1,53 @@
 import { mapActions, mapGetters } from 'vuex'
-import BaseButton from '@/components/BaseButton'
-import BaseInput from '@/components/BaseInput'
-import BaseTextArea from '@/components/BaseTextArea'
+import ListItem from '@/components/list/ListItem'
+import ModalDeleteConfirmation from '@/components/modals/ModalDeleteConfirmation'
+import InfiniteLoading from 'vue-infinite-loading'
 
 export default {
   name: 'QuestionBankDetail',
   components: {
-    BaseInput,
-    BaseTextArea,
-    BaseButton
+    ListItem,
+    ModalDeleteConfirmation,
+    InfiniteLoading
   },
   data () {
     return {
       questionBankDetail: {
         title: '',
         description: ''
       },
-      editMode: false
+      questions: [],
+      paging: {
+        page: 1,
+        size: 10,
+        totalRecords: 0
+      },
+      editMode: false,
+      showDeleteConfirmationModal: false,
+      state: '',
+      selectedId: '',
+      infiniteId: +new Date()
     }
   },
   created () {
     this.initPage()
   },
   computed: {
     ...mapGetters([
-      'questionBank'
+      'questionBank',
+      'questionList',
+      'accessList'
     ]),
-    cancelButtonText () {
-      return this.editMode ? 'Cancel' : 'Return'
-    },
-    actionButtonText () {
-      return this.editMode ? 'Save' : 'Edit'
+    deleteModalMessage() {
+      return !!this.selectedId ? 'Are you sure you want to delete this question?' : 'Are you sure you want to delete this question bank?'
```

Comment:
is `!!` necessary?

Reply:
Yes, `!!` is used to handle _truthy falsy value_ in javascript. CMIIW